addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,33 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {mail.messaging}
+         */
+        _computeMessaging() {
+            return [['link', this.env.messaging]];
+        }
+
+        /**
+         * @private
+         * @returns {String[]}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            // The goal here is to target small screen size and not actual mobile device
+            // to allow the sending of message with shortcut.
+            if (
+                (this.thread.model === 'mail.channel' && this.thread.mass_mailing) ||
+                (this.env.messaging.device.isMobile && this.thread.chatWindow)
```

Comment:
Is the condition on `chatWindow` really necessary? I think the fact that only chat window is concerned by the bug is "bad luck", but not especially a condition we actually want to enforce?